### PR TITLE
Fix spelling mistakes

### LINKED ISSUE #6400: Minor spelling errors
### What steps are needed to reproduce the bug?

Read the misspellings reported at https://github.com/jsoref/IoT-For-Beginners/commit/152f85d8f8ced0c5b63649a5e9fb622ee2d36705#commitcomment-86437888

### What Stylelint configuration is needed to reproduce the bug?

N/A

### How did you run Stylelint?

M/A

### Which version of Stylelint are you using?

main

### What did you expect to happen?

No [spelling errors](https://github.com/jsoref/IoT-For-Beginners/actions/runs/3226549594/jobs/5280202212)

### What actually happened?

https://github.com/jsoref/IoT-For-Beginners/commit/152f85d8f8ced0c5b63649a5e9fb622ee2d36705#commitcomment-86437888

### Does the bug relate to non-standard syntax?

_No response_

### Proposal to fix the bug

_No response_

### PR DESCRIPTION
**Pull request by jsoref**
> Which issue, if any, is this issue related to?

Closes #6400

> Is there anything in the PR that needs further explanation?

This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/IoT-For-Beginners/commit/152f85d8f8ced0c5b63649a5e9fb622ee2d36705#commitcomment-86437888

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/stylelint/commit/bd2b632047e1b0d9e707e8ed7af4f2b7d50c642a
 
Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1140,7 +1140,7 @@ Changes:
 
 ## 7.0.0
 
-- Removed: `--extract` and `extractSyleTagsFromHtml` options. Instead, [build](/docs/developer-guide/processors.md) and [use](/docs/user-guide/configure.md#processors) processors.
+- Removed: `--extract` and `extractStyleTagsFromHtml` options. Instead, [build](/docs/developer-guide/processors.md) and [use](/docs/user-guide/configure.md#processors) processors.
```
Comment (jsoref) [inline CHANGELOG.md:1143]: https://github.com/stylelint/stylelint/commit/e595dad4cf8609910a0a05b01fb93a0ab1c3f4a3

File: lib/formatters/__tests__/jsonFormatter.test.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ describe('jsonFormatter', () => {
 		const results = [
 			{
 				...resultClean,
-				_privateVaribale: [],
+				_privateVariable: [],
```
Comment (jsoref) [inline lib/formatters/__tests__/jsonFormatter.test.js:27]: ?

File: lib/rules/color-hex-case/__tests__/index.js [hunk 3]
```diff
@@ -15,7 +15,7 @@ const sharedTests = {
 			description: 'ignore sass-like interpolation',
 		},
 		{
-			code: 'a { stroke: url(#gradiantA) }',
+			code: 'a { stroke: url(#gradientA) }',
```
Comment (jsoref) [inline lib/rules/color-hex-case/__tests__/index.js:18]: I'm pretty sure nothing cares

File: lib/rules/selector-pseudo-class-no-unknown/__tests__/index.js [hunk 4]
```diff
@@ -109,7 +109,7 @@ testRule({
 		{
 			code: 'a:defined { }',
 			description:
-				'represents any element that has been defined; includes any standard element built in to the browser, and custom elements that have been successfully defined (i.e. with the CustomElementRegistry.define() method).',
+				'represents any element that has been defined; includes any standard element built into the browser, and custom elements that have been successfully defined (i.e. with the CustomElementRegistry.define() method).',
```
Comment (jsoref) [inline lib/rules/selector-pseudo-class-no-unknown/__tests__/index.js:112]: See [1.b.](https://www.merriam-webster.com/dictionary/built-in)

File: lib/utils/__tests__/findAnimationName.test.js [hunk 5]
```diff
@@ -19,7 +19,7 @@ it('findAnimationName', () => {
 			value: 'INHERIT',
 		},
 	]);
-	expect(findAnimationName('3s @varialbe')).toEqual([]);
+	expect(findAnimationName('3s @variable')).toEqual([]);
```
Comment (jsoref) [inline lib/utils/__tests__/findAnimationName.test.js:22]: ?

**Review by jsoref — COMMENTED** [review]
Spelling corrections were generally suggested by Google Sheets. All fault mine.

**Review by jeddy3 — APPROVED** [review]
@jsoref Thank you. These all LGTM.

The changes to the tests themselves (rather than just the descriptions) are fine as they don't have any impact on them.

You're welcome to make further correction pull requests like this without opening an issue beforehand.

**Pull request merged**